text/0000-rfc-process-update.md

```diff
@@ -0,0 +1,166 @@
+
+### Improve the lifecycle of RFCs
+
+To address the triaging and inactivity problem of RFCs, I introduce the concept of FCP to close.
+
+Closing an RFC should be viewed as another triaging tool, not as a rejection of the RFC.
+Sometimes a rewrite of an RFC would be so fundamental that it would benefit of a fresh discussion in a new thread.
+Sometimes the original author is no longer active (Champions should help here as well),
+and someone else might want to take over the work in a new RFC.
+Sometimes the timing might not be right, or the feature might have been addressed some other way, and yes,
+sometimes it might be something that is not aligned with the team's values for the project.
+
+A good example of this is the [Named Blocks RFC](https://github.com/emberjs/rfcs/blob/master/text/0226-named-blocks.md#motivation),
+which lists in the motivation section previous attempts at similar ideas.
+
+Like the FCP to merge process, once an RFC is marked as FCP to close there will be a period of one week where people can raise new concerns.
+After that period of one week, the respective team will review and close the RFC or extend the period for another week.
+
+### Merge ember-cli/rfcs into emberjs/rfcs
+
+Part of removing confusion about the process is having a central repository for all Ember RFCs.
+
+To achieve merging ember-cli/rfcs into emberjs/rfcs the following will be done:
+
+* Remove active/completed concepts from ember-cli/rfcs
+  * Add front matter
+  * Copy both active and completed RFC files into `text` of emberjs/rfcs
+* Transfer active PRs and Issues to emberjs/rfcs
+* Archive ember-cli/rfcs
+
+There are some concerns about links breaking when we move the files to emberjs/rfcs,
+but given the fact that ember-cli/rfcs had the concept of active/completed by moving the files into different folders,
+links were already being broken.
+
+The ember-cli/rfcs do not need name or numbering changes, as there is currently no duplicated name.
+Going forward, the numbering should be unified by virtue of having a single repository.
+
+### Open process to more types of RFCs
+
+To build on the previous point, the front matter that is to be added to RFCs will make it easier to have RFCs for other areas of the project,
+like the design of the web resources, Guides content, marketing content, or features for core addons.
+
+Given that Ember is [organized into teams](https://emberjs.com/team/), with each team being responsible for certain projects,
+the RFC front matter should be oriented towards those teams.
+
+A list of the teams and respective projects should be added to the instructions,
+possibly with the addition of per-team instructions on specifics of the project.
+Additional templates might be created as well, such a design work template.
+
+### Provide more guidance for post-RFC
+
+At the moment it is not clear for some contributors what happens once an RFC is merged.
+How and when should work to implement an RFC be done?
```

Comment:
Needs another sentence or two. Maybe just specifying that the instructions will be added to a CONTRIBUTING.md file or something. It would be good to see a draft and link it here.